peercrawler_html_server.py

```diff
@@ -12,24 +12,25 @@
 import pynanocoin
 from acctools import to_account_addr
 from representative_mapping import representative_mapping
-from _logger import setup_logger, get_logger
+from _logger import setup_logger, get_logger, get_logging_level_from_int
 
 
 app = Flask(__name__, static_url_path='/peercrawler')
 logger = get_logger()
 
 ctx = pynanocoin.livectx
```

Comment:
The ctx must be chosen according to the comamdn lines flags, see how it set on other scripts.